packages/jest-reporters/src/GithubActionsReporter.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import type {AggregatedResult, TestResult} from '@jest/test-result';
+import BaseReporter from './BaseReporter';
+import type {Context} from './types';
+
+const newLine = /\n/g;
+const encodedNewLine = '%0A';
+const lineAndColumnInStackTrace = /^.*?:([0-9]+):([0-9]+).*$/;
+
+export default class GithubActionsReporter extends BaseReporter {
+  onRunComplete(
+    _contexts?: Set<Context>,
+    aggregatedResults?: AggregatedResult,
+  ): void {
+    if (!process.env.GITHUB_ACTIONS) {
```

Comment:
I think we should use `ci-info` instead of manually checking for the env. should make the test cleaner as well as you can use `jest.mock`

Reply:
fc6dd305ac